### PR TITLE
Add goimports check and fix import order for all files

### PR DESCRIPTION
**Pull request by songy23**
Updates #155.

### CODE REVIEW

**Comment by pjanotti** [comment]
Missing `goimports` on the tools to be installed.

```
Makefile:96: recipe for target 'goimports' failed
make: *** [goimports] Error 1
The command "make travis-ci" exited with 2.
```

**Comment by songy23** [comment]
Seems there's another error on import: 
https://travis-ci.org/open-telemetry/opentelemetry-service/builds/565729405?utm_source=github_status&utm_medium=notification
```bash
goimports FAILED => fix the import order in the following files:
exporter/zipkinexporter/zipkin.go
Makefile:96: recipe for target 'goimports' failed
make: *** [goimports] Error 1
```

**Comment by songy23** [comment]
Hmmm still the same error, though the exact same check passed on my desktop...

**Comment by pjanotti** [comment]
Have you updated the go.mod with the correct version of goimports? Likely your desktop is using a different version than travis.

**Comment by songy23** [comment]
Finally fixed in https://github.com/open-telemetry/opentelemetry-service/pull/209/commits/394ea591f8708c0dff7b58de43ebf2cf257fae3d...

File: Makefile [hunk 1]
```diff
@@ -32,8 +33,8 @@ all-srcs:
 
 .DEFAULT_GOAL := addlicense-fmt-vet-lint-test
```
Comment (tigrannajaryan) [inline Makefile]: Shouldn't this be updated to `addlicense-fmt-vet-lint-goimports-test`?

File: Makefile [hunk 2]
```diff
@@ -90,15 +91,28 @@ lint:
 	    echo "Lint finished successfully"; \
 	fi
 
+.PHONY: goimports
+goimports:
+	@IMPORTSOUT=`$(GOIMPORTS) -d . 2>&1`; \
+	if [ "$$IMPORTSOUT" ]; then \
```
[the remark this answers is not in the document]
Comment (songy23) [reply Makefile]: From reading the code, it seems goimports diff won't set an error exit code:
https://github.com/golang/tools/blob/0133cac3176f225883c5d817146de8633ed07ebc/cmd/goimports/goimports.go#L311-L336

I did an experiment and the result supported it:
```bash
$ goimports -d -e .
# some output
$ echo $?
0
```

`Diff` will only print output when there's an actual difference so I think it's safe to use.

File: Makefile (hunk 1 above)
Comment (songy23) [reply Makefile]: Good catch, fixed.

**Comment by songy23** [comment]
Need one more approval for this PR - PTAL.

**Pull request merged**